src/components/FilterSelector/FilterSelector.jsx

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from "react";
+import { useFilter } from "../../contexts/Filter/FilterContext";
+import { useNote } from "../../contexts/Note/NoteContext";
+import styles from "./FilterSelector.module.css";
+
+export default function FilterSelector() {
+  const [filterForm, setfilterForm] = useState(false);
+
+  // const handlerFunction = (event) => {
+  //   const { value } = event.target;
+  //   setOption(value);
+  // };
+  const { filterDispatch, filterState } = useFilter();
+  const { filterByPriority } = filterState;
+  const taglist = [
+    {
+      title: "Todo",
+      color: "var(--component-blue-01)",
+    },
+    {
+      title: "Work",
+      color: "var(--component-green-01)",
+    },
+    {
+      title: "Chore",
+      color: "var(--component-yellow-01)",
+    },
+    {
+      title: "Shopping",
+      color: "var(--component-pink-01)",
+    },
+  ];
+
+  const pallete = [
+    { title: "WHITE", value: `var(--white)` },
+    { title: "BLUE", value: `var(--component-blue-02)` },
+    { title: "RED", value: `var(--component-red-03)` },
+    { title: "GREEN", value: `var(--component-green-02)` },
+    { title: "YELLOW", value: `var(--component-yellow-04)` },
+  ];
```

Comment:
Can shift these pieces of code to a new file meant for the static data!!

Reply:
Noted. Will be adding this to a constants file.